llama_index/response_synthesizers/generation.py

```diff
@@ -1,8 +1,9 @@
-from typing import Any, Optional, Sequence
+from typing import Any, Optional, Sequence, Tuple
 
 from llama_index.indices.service_context import ServiceContext
 from llama_index.prompts import BasePromptTemplate
 from llama_index.prompts.default_prompts import DEFAULT_SIMPLE_INPUT_PROMPT
+from llama_index.prompts.mixin import PromptDictType, PromptMixinType
 from llama_index.response_synthesizers.base import BaseSynthesizer
```

Comment:
(I must be looking at a bad commit) Don't we need to expose `simple_template` here?